examples/19-EnsembleModel-examples.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        regression_train_n_points=24,
</pre><p>we could use <code>regression_trian_n_points=-1</code> and <code>train_using_historical_forecasts=True</code> . Might yield better results?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>I tried a few settings, I found something acceptable (without using <code>regression_trian_n_points=-1</code> and <code>train_using_historical_forecasts=True</code>) that should be enough for the sake of this example notebook</p>